gamestonk_terminal/common/behavioural_analysis/reddit_model.py

```diff
@@ -0,0 +1,530 @@
+"""Reddit Model"""
+__docformat__ = "numpy"
+
+from typing import List, Dict, Tuple
+from datetime import datetime, timedelta
+import pandas as pd
+from prawcore.exceptions import ResponseException
+from requests import HTTPError
+from psaw import PushshiftAPI
+import praw
+import finviz
+from gamestonk_terminal import config_terminal as cfg
+
+from gamestonk_terminal.common.behavioural_analysis.reddit_helpers import find_tickers
+
+l_sub_reddits = [
```

Comment:
True.  Although these shouldn't slow down at all.  Just defined outside since it is used across most functions.  Could always just import them from a file instead, but that seems round about.

Reply:
The thing is that the way this work is that is runtime code. Meaning that once we start terminal we run through all imports and variables definition. Then let's say the user goes into Fundamentals and then quits. He'll never use this variables being declared, hence adding that latency was unnecessary.

The other side of the coin is if the user uses multiple functions that rely on these variable, then we add latency by re-declaring it multiple times.

I mean this is negligible stuff for now, but it adds up. I think that the "screener" ones that I've added should follow this because they are rather big.